src/ui/profile_preferences.ui

```diff
@@ -225,12 +225,12 @@
          <layout class="QGridLayout" name="gridLayout_groupBox_miscellaneous">
           <item row="0" column="0">
            <widget class="QCheckBox" name="mAlertOnNewData">
-            <property name="text">
```

Comment:
This change is unnecessary - it is just Git noise that has no effect so can be left out (undone in another commit to this PR).